webrtc.html

```diff
@@ -5763,6 +5763,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
              DOMString      mimeType;
              unsigned long  clockRate;
              unsigned short channels = 1;
+             unsigned long  maxptime;
+             unsigned long  ptime;
```

Comment:
I like matching ORTC, but something to note is that since "a=ptime" and "a=maxptime" aren't codec-specific attributes, these will be the same for all sent codecs. That's just a limitation of SDP. It may be nice to document this somewhere.